modules/search/class.jetpack-search.php

```diff
@@ -874,7 +874,15 @@ public function convert_wp_es_to_es_args( array $args ) {
 
 		$args = wp_parse_args( $args, $defaults );
 
-		$parser = new Jetpack_WPES_Search_Query_Parser( $args['query'], array( get_locale() ) );
+		/**
+		 * Filter the languages used by Jetpack Search's Query Parser.
+		 * @module search
+		 *
+		 * @since  7.9.0
+		 *
+		 * @param array $languages The array of languages. Default is value of get_locale().
+		 */
+		$parser = new Jetpack_WPES_Search_Query_Parser( $args['query'], apply_filters( 'jetpack_search_query_languages', array( get_locale() ) ) );
```

Comment:
What do you think about moving the docblock to right before the `apply_filters` call, to make sure the parser can catch it? I'm not 100% sure this is necessary, but it seems cleaner that way?

Reply:
https://github.com/Automattic/jetpack/pull/13630/commits/a06965484bb92f9974a2af4f633923ea97cec8c5